crypto/key_manager.go

```diff
@@ -144,7 +144,7 @@ func parsePrivateKey(key []byte) (crypto.PrivateKey, trillian.SignatureAlgorithm
 	// Our two ways of reading keys are ParsePKCS1PrivateKey and ParsePKCS8PrivateKey.
 	// And ParseECPrivateKey. Our three ways of parsing keys are ... I'll come in again.
 	if key, err := x509.ParsePKCS1PrivateKey(key); err == nil {
-		return key, trillian.SignatureAlgorithm_ANONYMOUS, nil
+		return key, trillian.SignatureAlgorithm_RSA, nil
```

Comment:
OK I was more asking if there are going to be cases where we need to know what the algorithm is but we can't tell. This might be something to address later.